py/src/scripts/load_data.py

```diff
@@ -37,29 +37,25 @@ def load_up_db(json_lst):
           feed_url=series_json.get('feedUrl'),
           genres=series_json.get('genres')
       )
-      commit_model(new_series)
-    except Exception:
-      print 'Error saving series'
 
-    # Episode work
-    episode_jsons = element['episodes']
-    try:
+      # Episode work
+      episode_jsons = element['episodes']
       new_eps = []
       for ep_json in episode_jsons:
         new_eps.append(Episode(
             title=ep_json.get('title'),
             author=ep_json.get('author'),
-            summary=ep_json.get('summary'),
-            pub_date=datetime.datetime.fromtimestamp(ep_json.get('pubDate')),
+            summary=ep_json.get('summary').encode('utf-8'),
+            pub_date=datetime.\
+              datetime.fromtimestamp(float(ep_json.get('pubDate'))),
             duration=ep_json.get('duration'),
             audio_url=ep_json.get('audioUrl'),
             tags=ep_json.get('tags'),
             series_id=ep_json.get('seriesId')
         ))
-      commit_models(new_eps)
+      commit_models([new_series] + new_eps)
```

Comment:
The list concat operation is pretty slow I think, it would be faster to make a `new_models` list initialized to `[new_series]` then append all the new episodes and commit that list